_routes/post_interactions.md

```diff
@@ -111,6 +111,39 @@ POST /api/v1/posts/:post_guid/hide
 Status: 204 No Content
 ~~~
 
+## Vote on poll
+
+Record a vote on a poll embedded into a post.
+
+~~~
+POST /api/v1/posts/:post_guid/vote
+~~~
+~~~json
+{
+  "poll_answer_id": 3
```

Comment:
Mhhhhh. Most of the time, we don't suffix `_id` and such things. In [`POST /api/v1/aspects/:aspect_id/contacts`](https://diaspora.github.io/api-documentation/routes/contacts.html#add-a-user-to-an-aspect), the field is named `person`, not `person_guid`. I see the benefit of suffixing with `_id`, but I also like the idea of having everything somewhat consistent, which means this should be `poll_answer` instead.

I'm undecided. What do you think?

Reply:
I really don't care either way, this is following the current implementation, so it was one PR less for me :D

Reply:
Hm. Let's make this `poll_answer` in a possibly futile attempt to keep things consistent. Sorry for the extra PR work this will result in.

Reply:
No worries, give me 3 minutes :D